eng/versioning/version_client.txt

```diff
@@ -105,6 +104,7 @@ com.azure:azure-quantum-jobs;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-search-documents;11.3.2;11.4.0-beta.3
 com.azure:azure-search-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-security-attestation;1.0.0-beta.1;1.0.0-beta.2
+com.azure:azure-security-confidentialledger;1.0.0-beta.1;1.0.0-beta.2
```

Comment:
I'd make this `1.0.0-beta.2` for both the dependency and current version given that if anything attempts to use this as a dependency it'll break.